clients/cypress.config.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
hm it's kind of weird this file showed up! do you know why that happened?

Reply:
Ah I don't think I need this file anymore, thanks!